Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -472,6 +515,15 @@ public double WindowTop()
             return top;
         }
 
+        public double WindowRight()
```

Comment:
what's this for?

Reply:
for Right (screen) position. it using in top-right option.